getting an error while trying to open the load window

Debian Testig x64

```
bin/Briss-2.0                                                                                                                                           [35/86]
Exception in thread "main" java.lang.NoClassDefFoundError: javafx/embed/swing/JFXPanel                                                             
        at at.laborg.briss.BrissGUI.<init>(BrissGUI.java:87)                                                                                       
        at at.laborg.briss.Briss.main(Briss.java:34)                                                                                               
Caused by: java.lang.ClassNotFoundException: javafx.embed.swing.JFXPanel                                                                           
        at java.base/jdk.internal.loader.BuiltinClassLoader.loadClass(BuiltinClassLoader.java:581)
        at java.base/jdk.internal.loader.ClassLoaders$AppClassLoader.loadClass(ClassLoaders.java:178)         
        at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:521)       
        ... 2 more                                                                                                                                 
Exception in thread "AWT-EventQueue-0" java.lang.NoClassDefFoundError: javafx/application/Platform
        at at.laborg.briss.BrissGUI.showOpenFileDialog(BrissGUI.java:291)    
        at at.laborg.briss.BrissGUI.lambda$init$6(BrissGUI.java:192)                                                                               
        at java.desktop/javax.swing.AbstractButton.fireActionPerformed(AbstractButton.java:1967)
        at java.desktop/javax.swing.AbstractButton$Handler.actionPerformed(AbstractButton.java:2308)
        at java.desktop/javax.swing.DefaultButtonModel.fireActionPerformed(DefaultButtonModel.java:405)
        at java.desktop/javax.swing.DefaultButtonModel.setPressed(DefaultButtonModel.java:262)
        at java.desktop/javax.swing.plaf.basic.BasicButtonListener.mouseReleased(BasicButtonListener.java:279)
        at java.desktop/java.awt.Component.processMouseEvent(Component.java:6632) 
        at java.desktop/javax.swing.JComponent.processMouseEvent(JComponent.java:3342)                                                                                
        at java.desktop/java.awt.Component.processEvent(Component.java:6397)                                                                                          
        at java.desktop/java.awt.Container.processEvent(Container.java:2263)      
        at java.desktop/java.awt.Component.dispatchEventImpl(Component.java:5008)                                                                  
        at java.desktop/java.awt.Container.dispatchEventImpl(Container.java:2321)                                                                  
        at java.desktop/java.awt.Component.dispatchEvent(Component.java:4840)                                                                                         
        at java.desktop/java.awt.LightweightDispatcher.retargetMouseEvent(Container.java:4918)                                                                        
        at java.desktop/java.awt.LightweightDispatcher.processMouseEvent(Container.java:4547)                                                                         
        at java.desktop/java.awt.LightweightDispatcher.dispatchEvent(Container.java:4488)                                                                                                     
        at java.desktop/java.awt.Container.dispatchEventImpl(Container.java:2307)                                                                                                             
        at java.desktop/java.awt.Window.dispatchEventImpl(Window.java:2772)                                                                                                                   
        at java.desktop/java.awt.Component.dispatchEvent(Component.java:4840)                                                                                                                 
        at java.desktop/java.awt.EventQueue.dispatchEventImpl(EventQueue.java:772)                                                                                                            
        at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:721)                         
        at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:715)                         
        at java.base/java.security.AccessController.doPrivileged(Native Method)                                                                                                               
        at java.base/java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:85)                                                                  
        at java.base/java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:95)                                                                  
        at java.desktop/java.awt.EventQueue$5.run(EventQueue.java:745)                         
        at java.desktop/java.awt.EventQueue$5.run(EventQueue.java:743)                         
        at java.base/java.security.AccessController.doPrivileged(Native Method)                                                                                                               
        at java.base/java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:85)                                                                  
        at java.desktop/java.awt.EventQueue.dispatchEvent(EventQueue.java:742)                                                                                                                
        at java.desktop/java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:203)                                                                                     
        at java.desktop/java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:124)                                                                                        
        at java.desktop/java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:113)                                                                                     
        at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:109)                                                                                                 
        at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:101)                                                                                                 
        at java.desktop/java.awt.EventDispatchThread.run(EventDispatchThread.java:90)                                                                                                         
Caused by: java.lang.ClassNotFoundException: javafx.application.Platform                       
        at java.base/jdk.internal.loader.BuiltinClassLoader.loadClass(BuiltinClassLoader.java:581)                                                                                            
        at java.base/jdk.internal.loader.ClassLoaders$AppClassLoader.loadClass(ClassLoaders.java:178)                                                                                         
        at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:521)                                                                                                                    
        ... 37 more                            
Exception in thread "AWT-EventQueue-0" java.lang.NoClassDefFoundError: javafx/application/Platform                                                                                            
        at at.laborg.briss.BrissGUI.showOpenFileDialog(BrissGUI.java:291)                                                                                                                     
        at at.laborg.briss.BrissGUI.lambda$init$6(BrissGUI.java:192)                           
        at java.desktop/javax.swing.AbstractButton.fireActionPerformed(AbstractButton.java:1967)             

```

--- a/README.md
+++ b/README.md
@@ -1,8 +1,13 @@
 # Briss 2.0
 
 Briss 2.0 is intended to be a GUI Update for the Briss PDF cropping tool.
+It is based on Briss 0.9 which is located at sourceforge: http://sourceforge.net/projects/briss/
 
-Briss 2.0 is based on Briss 0.9 which is located at sourceforge: http://sourceforge.net/projects/briss/
+### Installation
+
+Currently Briss 2.0 is in alpha therefore some features are still missing (for example the page skip list).
+If you want to give it a try you can download the pre release from https://github.com/mbaeuerle/Briss-2.0/releases.
+You will need Java 11 or newer to run it.
 
 ### Things that are done by now
 - Small refinements on gui which improve the workflow

--- a/build.gradle
+++ b/build.gradle
@@ -1,8 +1,13 @@
-apply plugin: 'java'
-apply plugin: 'application'
+
+plugins {
+    id 'org.openjfx.javafxplugin' version '0.0.8'
+    id 'java'
+    id 'application'
+}
 
 mainClassName = 'at.laborg.briss.Briss'
 applicationDefaultJvmArgs = ["-Xms128m", "-Xmx1024m"]
+def javaFxVersion = '11'
 
 repositories {
     maven { url "http://clojars.org/repo" } // For jpedal
@@ -13,4 +18,19 @@ repositories {
 dependencies {
     compile group: 'com.itextpdf', name: 'itextpdf', version: '5.5.10'
     compile group: 'org.jpedal', name: 'jpedal-lgpl', version: '4.74b27'
+
+    runtimeOnly "org.openjfx:javafx-controls:$javaFxVersion:win"
+    runtimeOnly "org.openjfx:javafx-controls:$javaFxVersion:linux"
+    runtimeOnly "org.openjfx:javafx-controls:$javaFxVersion:mac"
+    runtimeOnly "org.openjfx:javafx-base:$javaFxVersion:win"
+    runtimeOnly "org.openjfx:javafx-base:$javaFxVersion:linux"
+    runtimeOnly "org.openjfx:javafx-base:$javaFxVersion:mac"
+    runtimeOnly "org.openjfx:javafx-swing:$javaFxVersion:win"
+    runtimeOnly "org.openjfx:javafx-swing:$javaFxVersion:linux"
+    runtimeOnly "org.openjfx:javafx-swing:$javaFxVersion:mac"
+}
+
+javafx {
+    version = javaFxVersion
+    modules = [ 'javafx.controls', 'javafx.base', 'javafx.swing' ]
 }

--- a/gradle/wrapper/gradle-wrapper.properties
+++ b/gradle/wrapper/gradle-wrapper.properties
@@ -1,6 +1,6 @@
-#Thu Aug 02 16:02:28 CST 2018
+#Wed Apr 15 18:44:16 CEST 2020
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-4.9-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-4.9-all.zip

